docs/intro.md

```diff
@@ -0,0 +1,29 @@
+# HLS Overview
```

Comment:
Might be good to add a DASH overview that's similar at some point, since this project supports that as well (not necessary in this PR, more of a note for the future).

Reply:
💯 yeah, at some point I hope to actually add DASH into the depths of the documentation we do have